packages/core/src/docdb/commands/addRegion.ts

```diff
@@ -26,6 +27,11 @@ export async function addRegion(node: DBClusterNode | DBGlobalClusterNode): Prom
     return telemetry.docdb_addRegion.run(async () => {
         let globalClusterName = undefined
 
+        if (node.cluster.StorageEncrypted) {
+            void vscode.window.showErrorMessage('Currently supported for unencrypted clusters only.')
```

Comment:
nit: Shall we advise the user to navigate to the AWS console to add a region for Encrypted clusters?

Reply:
I've removed the menu for this so it's no longer available, but left in the code.  We can add a link to the message if desired.